content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -26,87 +26,211 @@ Tracer settings can be configured with the following environment variables:
+**Default**: N/A<br>
+**Example**: `mysql:my-mysql-service-name-db,pg:my-pg-service-name-db`<br>
 Provide service names for each plugin. Accepts comma separated `plugin:service-name` pairs, with or without spaces.
 
 `DD_TRACE_DISABLED_PLUGINS`
-: A comma-separated string of integration names automatically disabled when the tracer is initialized. Environment variable only, for example, `DD_TRACE_DISABLED_PLUGINS=express,dns`.
+: **Configuration**: N/A<br>
+**Default**: N/A<br>
+**Example**: `DD_TRACE_DISABLED_PLUGINS=express,dns`<br>
+A comma-separated string of integration names automatically disabled when the tracer is initialized.
 
 `DD_TRACE_LOG_LEVEL`
-: **Default**: `debug`<br>
+: **Configuration**: `logLevel`<br>
+**Default**: `debug`<br>
 A string for the minimum log level for the tracer to use when debug logging is enabled, for example, `error`, `debug`.
 
-
-For more options including the programmatic configuration API, see the [API documentation][2].
+Flush Interval
+: **Configuration**: `flushInterval`<br>
+**Default**: `2000`<br>
+Interval in milliseconds at which the tracer submits traces to the agent.
+
+`DD_TRACE_PARTIAL_FLUSH_MIN_SPANS`
+: **Configuration**: `flushMinSpans`<br>
+**Default**: `1000`<br>
+Number of spans before partially exporting a trace. This prevents keeping all the spans in memory for very large traces.
+
+`DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP`
+: **Configuration**: N/A<br>
+**Default**: N/A<br>
+A regex to redact sensitive data from incoming requests' query string reported in the `http.url` tag (matches are replaced with `<redacted>`). Can be an empty string to disable redaction or `.*` to redact all query string. **WARNING: This regex executes for every incoming request on an unsafe input (url) so make sure you use a safe regex.** |
+
+`DD_TRACE_CLIENT_IP_HEADER`
+: **Configuration**: N/A<br>
+**Default**: N/A<br>
+Custom header name to source the `http.client_ip` tag from.
+
+DNS Lookup Function
+: **Configuration**: `lookup`<br>
+**Default**: `require('dns').lookup`<br>
+Custom function for DNS lookups when sending requests to the agent.
+
+`DD_TRACE_AGENT_PROTOCOL_VERSION`
+: **Configuration**: `protocolVersion`<br>
+**Default**: `0.4`<br>
+Protocol version to use for requests to the agent. The version configured must be supported by the agent version installed or all traces are dropped.
+
+`DD_PROFILING_ENABLED`
+: **Configuration**: `profiling`<br>
+**Default**: `false`<br>
+Whether to enable profiling.
+
+`DD_TRACE_REPORT_HOSTNAME`
+: **Configuration**: `reportHostname`<br>
+**Default**: `false`<br>
+Whether to report the system's hostname for each trace. When disabled, the hostname of the agent is used instead.
+
+Experimental Features
+: **Configuration**: `experimental`<br>
+**Default**: `{}`<br>
+Experimental features can be enabled all at once using boolean `true` or individually using key/value pairs. Contact Datadog to learn more about the available experimental features.
```

Comment:
As the other commenter suggested, it might be a good choice to keep this out of the official docs. If you decide to keep it in though, it should say
```suggestion
Experimental features can be enabled all at once using Boolean `true` or individually using key-value pairs. [Contact Support][4] to learn more about the available experimental features.
```
plus add the following link URL at the bottom of this file:

```
[4]: /help/
```

And update the confluence page that called "The public docs say contact support" with the information support should provide when someone contacts them about this.

Reply:
I'm very slightly -1 on the capital-B Boolean as JavaScript typically describes primitive types in all lowercase.

```js
typeof true === 'boolean'
```